.github/workflows/deploy-prod.yml

```diff
@@ -32,8 +32,8 @@ jobs:
           BLOCKCHAIN_EXPLORER_BTC: 'https://www.blockchain.com/btc/tx/'
           BLOCKCHAIN_EXPLORER_ETH: 'https://etherscan.io/tx/'
           BLOCKCHAIN_EXPLORER_EOS: 'https://bloks.io/transaction/'
-          PPP_ENV: 'prod'
-          SENTRY_DSN: 'https://0b768263dc6247a5a2d309e07f771528@o324628.ingest.sentry.io/4503898341703680'
+          PPP_ENV: 'eos'
```

Comment:
@arsenijesavic This is not good. On prod it should be `prod`. Actually best would be to add `${{vars.pppEnv}}` or something like that